tests/contrib/mcp/test_mcp_llmobs.py

```diff
@@ -63,6 +64,23 @@ def test_llmobs_mcp_client_calls_server(mcp_setup, mock_tracer, llmobs_events, m
         tags={"service": "mcptest", "ml_app": "<ml-app-name>"},
     )
 
+    # asserting the remaining spans
+    assert llmobs_events[0] == _expected_llmobs_non_llm_span_event(
+        all_spans[0], span_kind="workflow", input_value=mock.ANY, tags={"service": "mcptest", "ml_app": "<ml-app-name>"}
+    )
+
+    assert llmobs_events[1] == _expected_llmobs_non_llm_span_event(
+        all_spans[1], span_kind="task", output_value=mock.ANY, tags={"service": "mcptest", "ml_app": "<ml-app-name>"}
+    )
+
+    assert llmobs_events[4] == _expected_llmobs_non_llm_span_event(
```

Comment:
not entirely sure why the `list_tools` appears _after_ the actual tool calls, but i did not observe this in local testing and is likely due to something in how our tests are set up